R/hp_spacyr.R

```diff
@@ -1,28 +1,80 @@
 #' Create a hierplane object using spacyr
 #'
-#' Creating a hierplane object from a string is possible by using the `spacyr`
-#' package. More specifically, we rely on `spacyr::spacy_parse()` to tokenize and
-#' tag the string provided. Note that this functionality requires the `spacyr`
-#' package.
+#' Creating a hierplane object from text is made possible by employing
+#' the `spacyr` package. More specifically, the `spacyr::spacy_parse()` function
+#' is used to tokenize and tag the sentence(s) provided. Note that this
+#' functionality requires the `spacyr` package.
 #'
-#' @param .data A string of text.
-#' @param settings Assign your dataframes columns to hierplane variables.
+#' If more than one sentence is detected in the input text, a list of hierplane
+#' objects is returned.
+#'
+#' @param .data A sentence.
+#' @param node_type `spacyr::spacy_parse()` variable to be used as node type (used for plane styling).
+#' @param link `spacyr::spacy_parse()` variable to be used to generate lane tags (i.e. connections between planes).
+#' @param attributes `spacyr::spacy_parse()` variable(s) to be used for generating the attribute tags.
+#' See full list of available attributes here: [](https://spacy.io/api/token#attributes)
 #' @param styles Assign styles to hierplane.
-#' @param ... Additional parameters to pass to `spacyr::spacy_parse`
+#' @param ... Additional parameters to pass to `spacyr::spacy_parse()`
+#'
+#' @examples
+#' if (FALSE) {
+#'
+#'   # Custom attribute tags
+#'   hp_spacyr("I was in college 25 years ago.", attributes = c("pos", "is_stop")) %>%
+#'     hierplane()
+#'
+#'   # Multiple sentences in input
+#'   hp_spacyr("I have a cat. Her name is mocha. She has a round belly.") %>%
```

Comment:
Best example

Reply:
We should require that all examples in r4fun packages to involve pets and animals